prime-router/src/main/kotlin/Report.kt

```diff
@@ -295,6 +295,13 @@ class Report {
         return table.getString(row, column)
     }
 
+    fun getStringByHl7Field(row: Int, hl7Field: String): String? {
+        val column = schema.elements.filter { it.hl7Field.equals(hl7Field, ignoreCase = true) }.firstOrNull()
+            ?: return null
+        val index = schema.findElementColumn(column.name) ?: return null
+        return table.getString(row, index)
+    }
+
```

Comment:
Allows me to pull information from a single row by the HL7 spec. Fails safely if there's no matching field for the spec and returns null. I will probably need to create a second one that searches the `hl7OutputFields` collection, and right now this just does a `firstOrNull()` and we could probably do a check to find any with a value and return that.  Not sure.